packages/core/src/shared/env.ts

```diff
@@ -21,6 +21,7 @@ export const ENV = {
   MAILCHIMP_SERVER_PREFIX: process.env.MAILCHIMP_SERVER_PREFIX as string,
   MEMBER_PROFILE_URL: process.env.MEMBER_PROFILE_URL as string,
   REDIS_URL: process.env.REDIS_URL as string,
+  RESOURCES_CHANNEL_ID: process.env.RESOURCES_CHANNEL_ID as string,
```

Comment:
@ramiAbdou @tomas-salgado I put this as a placeholder for now - i don't think a `#resources` (or something similar) channel exists yet for this feature